tests/modules/sightings/resources/test_custom_fields.py

```diff
@@ -108,16 +114,28 @@ def test_custom_fields_on_sighting(
             'updatedHouston': sighting.updated.isoformat() + '+00:00',
             'assets': assets,
             'featuredAssetGuid': str(sighting.featured_asset_guid),
-            'customFields': {cfd_id: cfd_test_value},
+            'customFields': {
+                cfd_id: cfd_test_value,
+                cfd_multi_id: cfd_multi_value,
+            },
             # Only asserting that these fields exist
             'time': full_sighting.json['time'],
             'timeSpecificity': full_sighting.json['timeSpecificity'],
             'createdEDM': full_sighting.json['createdEDM'],
         }
     )
 
+    # for some reason the assert above *passes* even though cfd_multi_value is wrong (maybe the >=)
```

Comment:
comment should say "even **when**".  that is to say, the above test passed when edm was returning the _wrong value_ for the multi-value.